File: BankApplicationAPI/BankApplicationAPI/Models/ComplaintStatusHistory.cs
namespace BankApplicationAPI.Models;

public partial class ComplaintStatusHistory
{
    public int StatusHistoryId { get; set; }

    public int? ComplaintId { get; set; }

    public DateTime? StatusDate { get; set; }

    public string? ComplaintStatus { get; set; }

    public string? StatusComments { get; set; }

    public virtual Complaint? Complaint { get; set; }
}

File: BankApp/src/app/Modules/ComplaintStatusHistory.ts
import { Complaint } from "./Complaint";

export interface ComplaintStatusHistory {
    statusHistoryId?: number | null;
    complaintId?: number | null;
    statusDate?: Date | null;
    complaintStatus?: string | null;
    statusComments?: string | null;
    complaint?: Complaint | null;
}